src/components/Contact/index.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+import "./index.css";
+
+export default class Contact extends Component {
+  static propTypes = {
+    children: PropTypes.node.isRequired,
+  };
+
+  render() {
+    const { children } = this.props;
+
+    return (
+      <a styleName="root" href="mailto:contact@sv-health.io">
```

Comment:
Sorry but I think you don't need a component for this... you can just use this directly

Reply:
I've created the component because we'll have the `copy to clipboard` feature

Reply:
Ok then!